### ISSUE TITLE
🙏🏻 Add support for editing reply markup of the message

### ISSUE THREAD

**Issue by mitinarseny** [tucnak/telebot#227]
Please, add support for editing only reply markup of given message with Telegram API method [editMessageReplyMarkup](https://core.telegram.org/bots/api#editmessagereplymarkup).  
So the method should look like:
```go
func (b *Bot) EditReplyMarkup(m *Message, [][]tb.InlineButton) (*Message, error) {
    // ...
}
```

**Comment by demget** [comment]
Already implemented it (#224)! Check my telebot repository, you can merge `patch-1` branch with your local repo.

**Comment by demget** [comment]
Actually, I've been trying to add this and other changes for a long time, but, as you can see, pull requests are accepted very rarely here...

**Comment by mitinarseny** [comment]
@demget mine was accepted two days ago...)
@tucnak can you take a look at this one, please? It would be really very useful!

**Comment by stek29** [comment]
Fixed in #224
